src/libs/actions/Report.js

```diff
@@ -727,15 +727,20 @@ function fetchAllReports(
                 return;
             }
 
-            // The string cast here is necessary as Get rvl='chatList' may return an int
-            reportIDs = String(response.chatList).split(',');
+            // The cast here is necessary as Get rvl='chatList' may return an int or Array
+            reportIDs = String(response.chatList)
+                .split(',')
+                .filter(_.identity);
 
             // Get all the chat reports if they have any, otherwise create one with concierge
-            if (reportIDs.length) {
+            if (reportIDs.length > 0) {
                 return fetchChatReportsByIDs(reportIDs);
             }
 
-            return fetchOrCreateChatReport([currentUserEmail, 'concierge@expensify.com']);
+            return fetchOrCreateChatReport([currentUserEmail, 'concierge@expensify.com'])
+                .then((createdReportID) => {
+                    reportIDs = [createdReportID];
+                });
```

Comment:
another option could be to have `fetchOrCreateChatReport()` return a reportID in an array and just use the `reportIDs` passed to the next chain. But this works (I had this idea from another PR I'm working on).

Reply:
This would be changing the other function for the convenience of the current usage.
I wouldn't expect to receive a list of one element from a function like `fetchOrCreateChatReport`